pandas/_libs/tslibs/timestamps.pyi

```diff
@@ -27,6 +27,7 @@ _DatetimeT = TypeVar("_DatetimeT", bound=datetime)
 def integer_op_not_supported(obj: object) -> TypeError: ...
 
 class Timestamp(datetime):
+    _reso: int
```

Comment:
This will become public eventually?

Reply:
Ah gotcha. Agreed since `resolution` is a public `datetime.datetime` attribute already